plutus-benchmark/nofib/test/formulaBudget.budget.golden

```diff
@@ -1,3 +1,3 @@
-({ cpu: 37605791908
-| mem: 161972648
+({ cpu: 26593253908
```

Comment:
I wonder why this one is improved so much compared to the others...

Reply:
> I wonder why this one is improved so much compared to the others...

@kwxm This is my understanding: the reason float-in reduces costs is because if a binding is floated into an unevaluated branch, it reduces the number of lambda abstractions and applications (which are what bindings are compiled into) as well as delays (because the arguments of the applications are usually inside `delay`s) the CEK machine needs to perform. This particular test case probably has many bindings floated into unevaluated branches. Here are the number of CEK steps:

| step kind | without float-in | with float-in |
| ------------- | ------------- | ------------- |
| BLamAbs  | 594154  | 434552  |
| BApply  | 496702  | 337100 |
| BDelay  | 195463  | 35861 |

The rest of the step kinds are identical between the two.